applications/GeoMechanicsApplication/tests/test_thermal_element/test_micro_climate_2/ProjectParameters.json

```diff
@@ -0,0 +1,209 @@
+    "newmark_theta"                      : 0.5,
+    "rayleigh_m"                         : 0.0,
+    "rayleigh_k"                         : 0.0,
+    "strategy_type"                      : "newton_raphson",
+    "convergence_criterion"              : "temperature_criterion",
+    "temperature_relative_tolerance"     : 1.0E-4,
+    "temperature_absolute_tolerance"     : 1.0E-9,
+    "residual_relative_tolerance"        : 1.0E-4,
+    "residual_absolute_tolerance"        : 1.0E-9,
+    "min_iterations"                     : 2,
+    "max_iterations"                     : 15,
+    "number_cycles"                      : 2,
+    "reduction_factor"                   : 1.0,
+    "increase_factor"                    : 1.0,
+    "desired_iterations"                 : 2,
+    "max_radius_factor"                  : 10.0,
+    "min_radius_factor"                  : 0.1,
+    "max_line_search_iterations"         : 5,
+    "first_alpha_value"                  : 0.5,
+    "second_alpha_value"                 : 1.0,
+    "min_alpha"                          : 0.1,
+    "max_alpha"                          : 2.0,
+    "line_search_tolerance"              : 0.5,
+    "rotation_dofs"                      : true,
+    "linear_solver_settings" : {
+      "solver_type"                      : "bicgstab",
+      "tolerance"                        : 1.0e-6,
+      "max_iteration"                    : 1000,
+      "scaling"                          : true,
+      "preconditioner_type"              : "ilu0"
+    },
+    "problem_domain_sub_model_part_list" : ["Soil1"],
+    "processes_sub_model_part_list"      : ["StartThermo","PointThermo","TopThermo","BotThermo"],
+    "body_domain_sub_model_part_list"    : ["Soil1"]
+  },
+  "output_processes" : {
+    "gid_output" : [{
+      "python_module"                    : "gid_output_process",
+      "kratos_module"                    : "KratosMultiphysics",
+      "process_name"                     : "GiDOutputProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.porous_computational_model_part",
+        "output_name"                    : "test_micro_climate_2_map",
+        "postprocess_parameters" : {
+          "result_file_configuration": {
+            "gidpost_flags" : {
+              "WriteDeformedMeshFlag"    : "WriteUndeformed",
+              "WriteConditionsFlag"      : "WriteElementsOnly",
+              "GiDPostMode"              : "GiD_PostAscii",
+              "MultiFileFlag"            : "SingleFile"
+            },
+            "file_label"                 : "step",
+            "output_control_type"        : "step",
+            "output_interval"            : 240,
+            "body_output"                : true,
+            "node_output"                : false,
+            "skin_output"                : false,
+            "plane_output"               : [],
+            "nodal_results"              : ["TEMPERATURE"],
+            "gauss_point_results"        : []
```

Comment:
I have not seen HEAT_FLUX output yet. It would be a natural thing to have, like we also have fluid flux.

Reply:
I created a draft issue on our product backlog to do this